backend/django/core/templates/projects/update/external_db.html

```diff
@@ -0,0 +1,119 @@
+{% extends "base.html" %}
+{% block page_title %}}Update Project Database Connection{% endblock %}
```

Comment:
Small thing, but there a bracket } shows up in the page title

Reply:
Great catch! Just updated